model/src/main/proto/spine/validation/rule.proto

```diff
@@ -68,6 +68,12 @@ message SimpleRule {
     // when the error is constructed.
     //
     string error_message = 5;
+
+    // If `true`, this rule is ignored when the field is not set.
+    //
+    // This does not apply to simple rules which are parts of a `CompositeRule`.
+    //
+    bool ignore_if_not_set = 6;
```

Comment:
Looking at it as at a `Message` property I would consider naming it `ignored_if_unset`.

"ignore_..." looks more like a command to me.